drivers/can/Kconfig.mspm0

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2024 Texas Instruments
+# SPDX-License-Identifier: Apache-2.0
+
+config CAN_MSPM0_CANFD
+	bool "MSPM0 CANFD driver"
+	default y
+	depends on DT_HAS_TI_MSPM0_CANFD_ENABLED
+	select CAN_MCAN
+	select USE_MSPM0_DL_MCAN
+	help
+	  Enable support for CANFD on the TI MSPM0 series.
+
+if CAN_MSPM0_CANFD
+
+config CAN_MAX_STD_ID_FILTER
```

Comment:
TODO, I guees 128 STD and 64 EXT are not possible at the same time, should a check be added.